server/server/controllers/Restaurant.js

```diff
@@ -198,9 +198,11 @@ async function removeMenuItem(req, res) {
 }
 
 async function getRestaurantInfo(req, res) {
+  const token = req.headers.authorization;
+  const {restaurantId} = req.params;
```

Comment:
We need to extract the restaurantId from the token instead of passing it in the params to succeed as authenticating the admin